matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/pushers/JsonPusher.kt

```diff
@@ -112,7 +114,19 @@ internal data class JsonPusher(
          * The default is false.
          */
         @Json(name = "append")
-        val append: Boolean? = false
+        val append: Boolean? = false,
+
+        /**
+         * Whether the pusher should actively create push notifications
+         */
+        @Json(name = "org.matrix.msc3881.enabled")
+        val enabled: Boolean = false,
```

Comment:
should this default to true? That would match the current behaviour when the field is omitted

Reply:
I missed this, it was only false for testing but it should be true. Good catch!